cont3xt/overview.js

```diff
@@ -281,6 +283,248 @@ class Overview {
+        field: 'firstSeen',
+        alias: 'PT PDNS First Seen'
+      },
+      {
+        type: 'linked',
+        from: 'PT Whois',
+        field: 'expiresAt',
+        alias: 'Domain Expiry date'
+      },
+      {
+        type: 'linked',
+        from: 'PT Whois',
+        field: 'contactEmail',
+        alias: 'PT contact email'
+      },
+      {
+        type: 'linked',
+        from: 'PT Whois',
+        field: 'organization',
+        alias: 'PT organization'
+      },
+      {
+        type: 'linked',
+        from: 'Threatstream',
+        field: 'Objects',
+        alias: 'ThreatStream events'
+      },
+      {
+        type: 'linked',
+        from: 'AlienVaultOTX',
+        field: 'Pulses:',
+        alias: 'OTX Pulses'
+      },
+      {
+        type: 'linked',
+        from: 'URLScan',
+        field: 'results',
+        alias: 'URLScan results'
+      },
+      {
+        type: 'linked',
+        from: 'VT Domain',
+        field: 'detected_urls',
+        alias: 'VT detected urls'
+      },
+      {
+        type: 'linked',
+        from: 'PT DNS',
+        field: 'results',
+        alias: 'PT PDNS'
+      }
+    ]
+  },
+  ip: {
+    name: 'IP_card',
+    title: 'IP Overview for %{query}',
+    fields: [
+      {
+        type: 'linked',
+        from: 'AbuseIPDB',
+        field: 'usageType',
+        alias: 'AbuseIPDB classification'
+      },
+      {
+        type: 'linked',
+        from: 'Spur',
+        field: 'infrastructure',
+        alias: 'Spur Infra:'
+      },
+      {
+        type: 'linked',
+        from: 'GreyNoise',
+        field: 'classification',
+        alias: 'GreyNoise classification'
+      },
+      {
+        type: 'linked',
+        from: 'BGPView',
+        field: 'Prefixes',
+        alias: 'BGPView prefixes'
+      },
+      {
+        type: 'linked',
+        from: 'Censys',
+        field: 'Services',
+        alias: 'Censys Open Port info'
+      },
+      {
+        type: 'linked',
+        from: 'Censys',
+        field: 'Certificates',
+        alias: 'Censys Certificates'
+      },
+      {
+        type: 'linked',
+        from: 'Shodan',
+        field: 'Service/Port info',
+        alias: 'Shodan Open Port info'
+      },
+      {
+        type: 'linked',
+        from: 'Shodan',
+        field: 'Certificates',
+        alias: 'Shodan Certificates'
+      },
+      {
+        type: 'linked',
+        from: 'AbuseIPDB',
+        field: 'Reports:',
+        alias: 'AbuseIPDB reports'
+      },
+      {
+        type: 'linked',
+        from: 'URLScan',
+        field: 'results',
+        alias: 'URLScan results'
+      },
+      {
+        type: 'linked',
+        from: 'VT IP',
+        field: 'detected_urls',
+        alias: 'VT detected urls'
+      },
+      {
+        type: 'linked',
+        from: 'Threatstream',
+        field: 'Objects',
+        alias: 'Threatstream tags'
+      }
+    ]
+  },
+  url: {
+    name: 'URL_card',
+    title: 'URL info for:  %{query}',
```

Comment:
There's an extra space before `%{query}` and this one has `:`. They should all have the `:` or none of them.